lib/MetaCPAN/Model/Search.pm

```diff
@@ -44,6 +44,16 @@ sub search_simple {
     return $results;
 }
 
+sub search_first {
```

Comment:
oh, that name was copied of search_simple & search_web.... it's basically search_simple but only returns the first element (so web doesn't have to dig through {hits}{hits} structures.

I don't mind changing it, any suggestions?